forch/faucetizer.py

```diff
@@ -69,13 +74,50 @@ def process_device_behavior(self, eth_src, behavior, static=False):
 
             self.flush_behavioral_config()
 
-    def process_faucet_config(self, faucet_config):
+    def _process_structural_config(self, faucet_config):
         """Process faucet config when structural faucet config changes"""
         with self._lock:
             self._structural_faucet_config = copy.copy(faucet_config)
 
+            self._next_cookie = 1
+            new_include = []
+            new_watched_acl_files = []
+            config_dir = os.path.dirname(self._structural_config_file)
+            for acl_file_name in self._structural_faucet_config.get('include', []):
+                acl_file_path = os.path.join(config_dir, acl_file_name)
+                self.reload_acl_file(acl_file_path)
+                new_include.append(self._augment_acl_file_path(acl_file_name))
+                new_watched_acl_files.append(acl_file_path)
+
+            self._structural_faucet_config['include'] = new_include
+
+            if not self._config.faucetize_interval_sec and self._reregister_acl_file_handlers:
+                self._reregister_acl_file_handlers(self._watched_acl_files, new_watched_acl_files)
+            self._watched_acl_files = new_watched_acl_files
+
             self.flush_behavioral_config()
 
+    def _process_acl_config(self, file_path, acls_config):
+        new_acls_config = copy.copy(acls_config)
+        if not self._next_cookie:
```

Comment:
why is this necessary?  Under what condition is _next_cookie not defined?

Reply:
With current code it is always set to 1 before being used... Just wanted to be able to notice corner cases during run time. Removing this check..